tests/conftest.py

```diff
@@ -29,7 +29,7 @@ def wrapper(*args, **kwargs):
 def make_ed25519_condition(public_key, *, amount=1):
     ed25519 = Ed25519Fulfillment(public_key=public_key)
     return {
-        'amount': amount,
+        'amount': str(amount),
```

Comment:
I know, this is just a conftest function, however:

I took a different approach regarding passing the amount. In Javascript, every amount now being passed to some function needs to be of type string, to avoid integer overflows on the user-side:


```python
# python max integer (within the range of asset.amount's max value)
>>> sys.maxint
9223372036854775807
```

```javascript
// js max integer (not within the range of asset.amount's max value)
Number.MAX_SAFE_INTEGER // 9007199254740991
```

Since Python's max integer is well within asset.amount's max value, we can still pass integers to user-facing functions, where as in Javascript we should always fall back to strings or even an extra library (e.g. big.js).


Just wanted to share that information.